flixel/FlxCamera.hx

```diff
@@ -1190,6 +1201,16 @@ class FlxCamera extends FlxBasic
 	}
 	
 	/**
+	 * Sets the filter array to be applied to the camera.
+	 * 
+	 * @param	filters
+	 */
+	public function setFilters(filters:Array<BitmapFilter>):Void
```

Comment:
To add a bit of coherence between the `FlxCamera` and `FlxGame` API. Read https://github.com/HaxeFlixel/flixel/pull/1635#discussion_r42324248